authorization_services/topics/policy-evaluation-api.adoc

```diff
@@ -5,8 +5,8 @@ When writing rule-based policies using JavaScript or JBoss Drools, {project_name
 
 This API consists of a few interfaces that provides you access to information such as:
```

Comment:
Found a couple minor issues in this bit that isn't part of your changes. Can you please fix this sentence when you do the other requested changes? :)

...few interfaces that provide you access to information, such as: